frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -149,6 +149,9 @@ function ReaderRolling:onSaveSettings()
     self.ui.doc_settings:saveSetting("inverse_reading_order", self.inverse_reading_order)
 end
 
+function ReaderRolling:onSaveSettings()
```

Comment:
It doesn't seem like a good idea to make this empty? If we need to make sure that settings are saved in onCloseDocument() then I'd propose either a simple 

```lua
function ReaderRolling:onCloseDocument()
    self:onSaveSettings()
end
```

or making sure the event is emitted from somewhere relevant.

But if I wanted to call save settings from somewhere I'd find this… unintuitive. ;-)